### ISSUE TITLE
Add Support for Detecting if Device was Rebooted

### ISSUE THREAD

**Issue by DavidBerdik** [libxposed/api#15]
As far as I am aware, there is currently no way to check if a device has been rebooted since a user enabled a module.

This can be an issue for modules that hook the System Framework since they require a reboot in order to function. The current approach to detecting if a module is active cannot distinguish between if a reboot has taken place or not, which could result in the module appearing to work properly even though it does not.

I would like to request that a function be added that returns false if there has not been a reboot since the module was enabled and true otherwise.

**Comment by vvb2060** [comment]
We are planning to provide target-side writable remote config, which we believe will satisfy a wider range of needs, including yours.

**Comment by DavidBerdik** [comment]
That's awesome! I look forward to seeing it! Thank you!

**Comment by DavidBerdik** [comment]
@Dr-TSNG Fair enough. Should this ticket be closed then? It seems like it should be open until this is addressed.
